src/connection-endpoint/websocket/text/text-protocol/message-builder.ts

```diff
@@ -213,8 +213,10 @@ const BUILDERS = {
  */
 export const getMessage = (message: Message, isAck: boolean = false): string => {
   if (!BUILDERS[message.topic] || !BUILDERS[message.topic][message.action]) {
-    console.log(message, isAck)
+    console.trace('missing builder for', message, isAck)
+    return ''
   }
+  
   const builder = BUILDERS[message.topic][message.action]
   if (!builder) {
```

Comment:
Could this check be removed since we are doing it and returning in the previous block?

Reply:
Done.